templates/support/index.html

```diff
@@ -187,7 +187,7 @@ <h4>Infrastructure</h4>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Security patching for more than 2,500 packages in Ubuntu Main repository (after 5 years of free LTS security patching)</li>
         <li class="p-list__item is-ticked">Up to 10 years of Linux support for Ubuntu LTS releases</li>
-        <li class="p-list__item is-ticked">Technical support for OpenStack, KVM, MAAS</li>
+        <li class="p-list__item is-ticked">Technical support for <a href='https://ubuntu.com/openstack'>OpenStack</a>, KVM, MAAS</li>
```

Comment:
We use relative URLs when it points to the current site. i.e. `/openstack` instead of `https://ubuntu.com/openstack`

Reply:
Thanks for reviewing! Fixed! :)